backoffice/.envs/local/.django

```diff
@@ -20,7 +20,7 @@ OPENSEARCH_HOST=es:9200
 OPENSEARCH_INDEX_PREFIX=backoffice-backend-local
 
 # Airflow
-AIRFLOW_BASE_URL=http://airflow-webserver:8070
+AIRFLOW_BASE_URL=http://airflow-webserver:8080
```

Comment:
wouldn't this be a conflict with ui? Hence we would not be able to run both inspirehep UI and airflow at the same time

Reply:
It doesnt because it's the container port (so internally) and not the exposed one. So from the outside you can reach ui under port 8080 and airflow under 8070